x-pack/plugins/security_solution/server/init_server.ts

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { IResolvers, makeExecutableSchema } from 'graphql-tools';
```

Comment:
feel free to remove `graphql`, `graphql-fields`, `graphql-tools` and all `apollo-*` packages from `package.json` 💪